definitions/ext-snowflake_warehouse/definition.yml

```diff
@@ -0,0 +1,22 @@
+# This telemetry is generated by this integration: https://github.com/newrelic/newrelic-snowflake-integration
+domain: EXT
+type: SNOWFLAKE_WAREHOUSE
+
+synthesis:
+  identifier: WAREHOUSE_NAME
+  name: WAREHOUSE_NAME
+  conditions:
```

Comment:
We only support one condition at the moment, and the `attribute` should be `eventType` as it references the event space in NRDB (which is SnowflakeVirtualWarehouse) rather than an attribute in the telemetry.